packages/cloud-react/lib/types.ts

```diff
@@ -10,6 +10,7 @@ export interface ComponentBase {
   children?: ReactNode;
   // passing custom styling.
   style?: CSSProperties;
+  ref?: MutableRefObject<AnyJson>;
```

Comment:
Would a generic HTML element be good for this?

``suggestion
  ref?: MutableRefObject<HTMLElement>;
```